<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>924</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="mBrowseButton">
      <property name="text">
       <string>browse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="mFilenameEdit">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>filename.csv</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Separator : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sepEdit">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="mProcessButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Process</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="mCurrentOpeLabel">
      <property name="text">
       <string>Current Operation :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="mOpProgress">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Logs :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="mLogsText">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>table name :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mtableNameEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Row size limit :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mLimitRowsEdit">
        <property name="text">
         <string>-1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="customEdit"/>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>766</width>
         <height>108</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>remove doubles</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="mGenerateId">
      <property name="text">
       <string>Generate id </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>use custom format</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
